lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -44,6 +44,14 @@ def get_ios_app_id_from_archive_plist(archive_path, plist_path)
         UI.shell_error!("can't extract GOOGLE_APP_ID") if identifier.empty?
         return identifier
       end
+
+      def blank?(value)
+        value.respond_to?(:empty) ? !!value.empty? : !value
+      end
+
+      def present?(value)
```

Comment:
This is unused. Did you want to include it for future use anyway? It does seem helpful, I know we've got some ugly presence-checking scattered around this codebase.

Reply:
Yeah just kept it in for future use

Reply:
SGTM